setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name='tc_prometheus',
-    version="1.0.0",
+    version="1.0.1",
```

Comment:
please leave this.. I will create a MR request which will "release" the new version, otherwise you are pushing me ;)